Library/Formula/seafile.rb

```diff
@@ -0,0 +1,108 @@
+class Seafile < Formula
+  desc "A daemon program used in seafile-client"
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/seafile/archive/v4.2.8.tar.gz"
+  sha256 "d9e3ed479fdc997df1b767133a065f2ceaa5a5a5f51184fe5964d01abc7c8cb2"
+
+  head "https://github.com/haiwen/seafile.git"
+
+  # FIX for homebrew autotools path
+  # FIX use system zlib
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!